lib/pages/chat_list/chat_list_view_style.dart

```diff
@@ -16,6 +16,7 @@ class ChatListViewStyle {
     return slidableSizeRatio * slidablesLength;
   }
 
-  static const Color pinSlidableColorRaw = Color(0xFF00C853);
-  static const Color readSlidableColorRaw = Color(0xFFAEAAAE);
+  static const EdgeInsets slidablePadding = EdgeInsets.all(4.0);
+  static const Color pinSlidableColorRaw = Color(0xFFAEAAAE);
```

Comment:
use linagoraSystemColor, because the designer have updated it

Reply:
please remember unread/read, pin/unpin have different colors.

Reply:
Hi I fixed colors and icon please check 


https://github.com/linagora/twake-on-matrix/assets/48354990/4706a6a0-385d-437b-910e-69122eb6e77f